lib/drivers/abbott/abbottFreeStyleLibre.js

```diff
@@ -263,7 +263,9 @@ export default function (config) {
       debug('processData: num post records:', data.post_records.length);
 
       if (data.post_records.length === 0) {
-        return cb(new Error('Device has no records to upload'), null);
+        const err = new Error('No records to upload');
```

Comment:
Should we add a `debug()` call above this line to be consistent with other devices?

Reply:
Done.